README.md

```diff
@@ -38,10 +38,25 @@ This template comes with:
 
 ## How to use with docker
 
-1. Have docker and docker-compose installed
-1. build and run the application with `docker-compose up`
-1. Run `docker-compose exec web bash -c "rails db:create db:migrate"`
-1. You can now try your REST services!
+1. Have `docker` and `docker-compose` installed (You can check this by doing `docker -v` and `docker-compose -v`)
+2. Modify the following lines in the `database.yml` file:
+  ``` yaml
+  default: &default
+    adapter: postgresql
+    encoding: unicode
+    pool: 5
+    username: postgres
+    password: postgres
+    host: db
+    port: 5432
+  ```
+3. Generate a secret key for the app by running `docker-compose run --rm --entrypoint="" web rake secret`, copy it and add it in your environment variables.
```

Comment:
you don't necessarily have to do this through docker-compose, but this works too

Reply:
you mean creating the secret?